docs/documentation.md

```diff
@@ -181,6 +181,39 @@ $options->mergeCells(0, 1, 0, 2, $writer->getCurrentSheet()->getIndex());
 $writer->close();
 ```
 
+## Row Grouping/Outline
+
+Rows can be grouped with the XLSX writers:
+
+```php
+use OpenSpout\Common\Entity\Row;
+use OpenSpout\Writer\XLSX\Writer;
+use OpenSpout\Writer\XLSX\Options;
+use OpenSpout\Writer\XLSX\RowAttributes;
+
+$options = new Options();
+
+// Required or the generated file will be corrupted
+// Looks like empty rows/cell cannot have `defaultRowHeight` equal to 0
+// which will be added when the setting is not set.
+$options->DEFAULT_ROW_HEIGHT = 15;
```

Comment:
Can we something about this, and not require the user to specify it?

Reply:
The problem is in empty rows which are required to split outline regions (or Excell will merge them into one region) - with no height we will have `<sheetFormatPr defaultRowHeight="0"/>` and (web-based) Excel[^1][^2]  will report about cryptic error "not in a range" (or something like...) and file will not be opened at all. I've tried to add the cell, the cell with empty inline string, but no luck :(

Looks like height required in any case 🤷‍♂️

[^1]: standalone version seems can open it, but with "file was recovered" message
[^2]: Libreoffice Calc open file without any warnings

Reply:
I don't like the idea that this library can produce a corrupted file if a particular option is not set.

Can you have a try to have a workaround within the code? I.E. if an emtpy row is added, the default row height gets set to a non zero value?

Reply:
> `<row r="5" outlineLevel="1" customHeight="1" ht="15"></row>`

Doesn't work. So I have no more ideas...

![image](https://user-images.githubusercontent.com/1329824/201336704-3ead6783-cd54-4c98-9940-d3aa4fcebe5c.png)